src/System.Collections/src/System/Collections/Generic/LinkedList.cs

```diff
@@ -188,7 +188,7 @@ public void CopyTo(T[] array, int index)
 
             if (index < 0 || index > array.Length)
```

Comment:
That would be a subjective question, with a subjective answer. I don't find it hurts readability, but then I'm used to it, so I'm probably a bad data-point! Other people might disagree strongly.